### PR TITLE
tweak what we look for in apprenticeship rubrics

### PR DESCRIPTION
**Pull request by borisonr**

### CODE REVIEW

File: interviews/frontend-takehome.md [hunk 1]
```diff
@@ -39,19 +39,17 @@ Non-core features encouraged for completion after the core features.
 - 👍 3: Requested delighter mostly completed or completed with a few minor bugs/flaws
 - 👎 2: Little to no delighter content was completed
 
-## Frontend Specifics
```
Comment (hmafzal) [inline interviews/frontend-takehome.md:42]: Line 17 has a link to this line. I think we may want to keep the heading?

File: interviews/backend-takehome.md [hunk 2]
```diff
@@ -38,3 +38,16 @@ Note that bonus points may be reduced if Solution Quality is not great.
 - ⭐️ 4: Requested delighter content completed fully.
 - 👍 3: Requested delighter mostly completed or completed with a few minor bugs/flaws.
 - 👎 2: Little to no delighter was completed.
+
+## What We Look For
```
Comment (hmafzal) [inline interviews/backend-takehome.md:42]: Likewise, maybe add a link to a  `## Backend Specifics` section to `### Backend Knowledge`, similar to how frontend one is set up

**Review by hmafzal — APPROVED** [review]
LGTM besides the linking comments

File: interviews/frontend-takehome.md (hunk 1 above)
Comment (borisonr) [reply interviews/frontend-takehome.md:42]: oh interesting, but frontend specifics doesn't make sense to me... does it make sense to you?

Comment (hmafzal) [reply interviews/frontend-takehome.md]: I think the intent is to give further explanation around what we look for or don't look for for frontend knowledge.

Maybe we could rename it to `Frontend Knowledge Specifics`

Comment (borisonr) [reply interviews/frontend-takehome.md]: hmm ok i guess it's fine as is

**Review by sylvanasantos — APPROVED** [review]
makes sense to me!

**Pull request merged**